docs/howto/architectures.rst

```diff
@@ -0,0 +1,234 @@
+Architectures
+=============
+
+By default, Snapcraft builds a snap to run on the same architecture as the build
+environment. This behaviour can be modified with the root keyword
+``architectures`` in the snap’s ``snapcraft.yaml``.
+
+The ``architectures`` keyword is used to create a build plan. See
+:ref:`build plans<build-plans>` for an explanation on how build plans are
+created.
+
+How to create a snap for a specific architecture
+------------------------------------------------
+
+To create a snap that will be built on ``amd64`` and built for ``amd64``, use
+one of the ``snapcraft.yaml`` snippets below.
+
+core22
+^^^^^^
+
+.. code-block:: yaml
+
+  architectures:
+    - build-on: amd64
+      build-for: amd64
```

Comment:
Should we change all examples to `[amd64]`?

Reply:
Yes, I updated all the examples and added a note.